teams.md/package.json

```diff
@@ -4,24 +4,24 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
-    "start:en": "docusaurus start --locale en",
-    "start:es": "docusaurus start --locale es",
-    "start:zh": "docusaurus start --locale zh-Hans",
+    "start": "concurrently --kill-others \"npm run generate:docs:watch\" \"docusaurus start\"",
+    "start:simple": "docusaurus start",
+    "prebuild": "npm run generate:docs",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
     "clear": "docusaurus clear",
     "serve": "docusaurus serve",
-    "write-translations": "docusaurus write-translations",
```

Comment:
wonder if there should be a script that would create the associated folders for a given file. `npm run scaffold <file_path>` or something